src/components/Schema/Property.tsx

```diff
@@ -123,13 +140,18 @@ const Property: React.FC<IProperty> = (props) => {
 
       {example && (
         <Example
+          label="Example"
           language={exampleLanguage}
         >
-          {formattedExample}
+          <>
+            {exampleLanguage === 'json' ? '//' : '#'}
+            {` JSON Path: ${schemaJsonPath}.${name}\n`}
```

Comment:
This should be `${schemaJsonPath}/${name}`.  We should also consider calling it `JSON Pointer` instead of `JSON Path`

Reply:
I think this is also causing top-level values to display as `/.key` instead of `/key`

Reply:
I've updated this logic with my latest push.